dev/io.openliberty.org.jboss.weld5/bnd.overrides

```diff
@@ -85,7 +85,7 @@ Export-Package: \
  org.jboss.weld.contexts.cache;version="5.1.0.Final",\
  org.jboss.weld.contexts.conversation;version="5.1.0.Final";uses:="jakarta.inject,org.jboss.weld.context,org.jboss.weld.manager",\
  org.jboss.weld.contexts.unbound;version="5.1.0.Final";uses:="jakarta.enterprise.context,jakarta.enterprise.context.spi,org.jboss.weld.context,org.jboss.weld.context.api,org.jboss.weld.contexts,org.jboss.weld.serialization.spi",\
- org.jboss.weld.event;version="5.1.0.Final";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject.spi,jakarta.enterprise.util,org.jboss.weld.bean,org.jboss.weld.bean.builtin,org.jboss.weld.bootstrap.api,org.jboss.weld.events,org.jboss.weld.injection,org.jboss.weld.manager,org.jboss.weld.module,org.jboss.weld.util",\
+ org.jboss.weld.event;version="5.1.0.Final";thread-context=true;uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject.spi,jakarta.enterprise.util,org.jboss.weld.bean,org.jboss.weld.bean.builtin,org.jboss.weld.bootstrap.api,org.jboss.weld.events,org.jboss.weld.injection,org.jboss.weld.manager,org.jboss.weld.module,org.jboss.weld.util",\
```

Comment:
https://github.com/OpenLiberty/open-liberty/pull/24636 removes the need for this line. It can/should be removed.